### ISSUE TITLE
interactive examples doesn't work when serving documentation using melos

### ISSUE THREAD

**Issue by immadisairaj** [flame-engine/flame#2079]
The issue arises when a contributor wants to work on interactive examples and tries to build the documentation locally using `melos doc-build-live` (or upcoming `melos doc-serve`)

# Current bug behaviour

When building using the above command, the interactive examples loaded in web give out "404 : Not Found" error (shown in below screenshot).

![Screenshot 2022-10-14 at 18 48 13](https://user-images.githubusercontent.com/40348358/195856725-0d6e9b7b-d94b-4778-80d0-6537c4021353.png)

This issue also comes when we try to use the `make livehtml` command.

# Expected behaviour

The interactive examples should work as expected without any error.

# Steps to reproduce

1. Go [here](https://docs.flame-engine.org/main/development/documentation.html#building-documentation-locally) and have the perquisites to build the documentation locally
2. Try to build the documentation using `melos doc-build-live` (or `melos doc-serve`)
3. Once the documentation is built, served and a browser opens, go to [effects](http://127.0.0.1:8000/flame/effects.html) section where there are a lots of interactive examples that are not running.

instead of 2 from above steps, `cd doc/_sphinx/` then `make livehtml` (internally same as the above commands) can also be used

# More information

The command `melos doc-build-live` will be changed to `melos doc-serve` once #2078 lands

The `make livehtml` is using [`sphinx-autobuild`](https://pypi.org/project/sphinx-autobuild/). Can be seen in [Makefile](https://github.com/flame-engine/flame/blob/a34f1df7904f0bd54fb8465265b24e21be0f4dc2/doc/_sphinx/Makefile#L16-L17). This might probably be causing the documentation to not serve properly.

I might not be currently working on this issue. Others can take this up.

**Comment by munsterlander** [comment]
I believe @ufrshubham and I have identified the issue. I assume you are on windows?  If so, I have the fix ready, I just need to verify one more thing and can submit the pr.

**Comment by immadisairaj** [comment]
> I assume you are on windows?

Nah, I am on macOS. As from what shubham and I discussed recently, windows uses "make.bat" and others use "Makefile" (which i am not completely sure).

The issue in windows is that `make livehtml` isn't working which is a separate issue.

**Comment by munsterlander** [comment]
Well my bad, lol

**Comment by spydon** [comment]
Does this work if you do `melos doc-build` before `melos doc-build-live`?

**Comment by immadisairaj** [comment]
> Does this work if you do `melos doc-build` before `melos doc-build-live`?

No, it doesn't. It only work when we do `melos doc-build` and then the python serve command
